packages/elements/src/component-factory-strategy.ts

```diff
@@ -71,78 +71,108 @@ export class ComponentNgElementStrategy implements NgElementStrategy {
    */
   private readonly unchangedInputs = new Set<string>();
 
-  constructor(private componentFactory: ComponentFactory<any>, private injector: Injector) {}
+  /** check for zone.js loaded */
+  private readonly zoneless: boolean;
+
+  /** service for setting zone context */
+  private readonly ngZone: NgZone;
```

Comment:
Let's move the initialization here:

```suggestion
  private readonly ngZone = this.injector.get<NgZone>(NgZone);
```